content/response-to-reviewers.md

```diff
@@ -0,0 +1,45 @@
+# Response to Reviewers
+
+The Deep Review ([preprint version 1](https://www.biorxiv.org/content/early/2017/05/28/142760)) was submitted _Journal of the Royal Society Interface_.
+We have now [received](https://github.com/greenelab/deep-review/issues/678) the peer reviews and are working on addressing the comments.
+When submitting a pull request to revise the Deep Review in response to reviewers, please also comment on the relevant section in this document.
+The comments should respond to the reviewer comments and should link to the pull requests or issues where they are addressed.
+
+After each reviewer criticism, there is a `TODO`.
+Replace the TODO with the relevant response.
+Feel free to also break up paragraphs as necessary, but remember to keep the reviews formatted as blockquotes.
+
+***
+
+## Referee 1
+
+> The authors summarized over 400 literature references purely by narratives. The authors provided synopsis for each important reference, but lacks synthesis of related work. It would be better to synthesize related work into a table and analyze their characteristics. 
+
+TODO
```

Comment:
For the ones that just say `todo`, do you want to add a link to the connected issue?